app/views/trainees/confirm_deferrals/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render PageTitle::View.new(title: "trainees.deferral_details.confirm") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: 'Back',
+    href: trainee_deferral_path(@trainee)
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <span class="govuk-caption-l">
+      <%= trainee_name(@trainee) %>
+    </span>
+    <h1 class="govuk-heading-l">Check deferral details</h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render Trainees::DeferralDetails::View.new(@trainee) %>
+
+    <%= form_with url: trainee_confirm_deferral_path(@trainee), method: :put, local: true do |f| %>
+      <%= hidden_field_tag :trainee_id, @trainee.id %>
```

Comment:
What's this hidden field tag for? You shouldn't need this I don't think.

Reply:
Thanks, removed.